index.js

```diff
@@ -112,6 +112,16 @@
     };
   }
 
+  //  clean :: Object -> Object
+  function clean(x) {
+    var o = Object.create(null);
+    for (var k in x) {
+      /* istanbul ignore else */
+      if (has(k, x)) o[k] = x[k];
+    }
+    return o;
+  }
+
```

Comment:
Unless you had a reason for inserting this function here, let's preserve alphabetical order. :)